db/seed.js

```diff
@@ -0,0 +1,43 @@
+var mysql = require('mysql');
+var createData = require('./seederFunction.js')
+
+var con = mysql.createConnection({
+    host: "localhost",
+    user: "root",
+    password: "password",
+    database: "tradeDeskdb" //leave out first run
+});
+
+con.connect(function(err) {
+    if (err) throw err;
+    console.log("Connected!");
+    let stockArray = ['AAPL']
+    
+    // con.query("CREATE DATABASE tradeDeskdb", function (err, result) {
```

Comment:
seems like this was left in, but no longer necessary. would consider removing it or turning it into a comment if you had a reminder to leave for yourself.